src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -372,7 +372,10 @@ public void HandleDatabaseRecordChange()
 
             if (newRecord == null)
             {
-                DropAllOutgoingConnections();
+                if (_log.IsInfoEnabled)
+                    _log.Info("Database record is not found, so we drop all outgoing connections.");
```

Comment:
Need to add info on _which_ connection is being dropped, and from what database.